Subscriber/Backend.php

```diff
@@ -128,6 +128,7 @@ public function onPostDispatchPayment(ActionEventArgs $args)
         $view->addTemplateDir($this->pluginDir . '/Resources/views/');
 
         if ($args->get('request')->getActionName() === 'load') {
+            $view->assign('deactivatedPaymentMethods', PaymentMethodProvider::getDeactivatedPaymentMethods());
```

Comment:
would it be easier to explode the array directly to a comma separated string? :thinking: so you do not need to en- and decode to JSON in ExtJs

Reply:
I dont know if this brings a benefit. Because you have to implode the string and explode the string to an array to.